### PR TITLE
Port procinfo

### PR DESCRIPTION
**Pull request by patryk4815**
procinfo from https://github.com/pwndbg/pwndbg/pull/2542

### CODE REVIEW

File: pwndbg/commands/procinfo.py [hunk 1]
```diff
@@ -66,12 +67,39 @@
...
+        data = pwndbg.aglib.file.get("/proc/net/tcp").decode()
+        return pwndbg.lib.net.tcp(data)
+
+    @classmethod
+    def unix(cls):
+        # We use errors=ignore because of https://github.com/pwndbg/pwndbg/issues/1544
+        # TODO/FIXME: this may not be the best solution because we may end up with
+        # invalid UDS data. Can this be a problem?
+        data = pwndbg.aglib.file.get("/proc/net/unix").decode(errors="ignore")
+        return pwndbg.lib.net.unix(data)
```
Comment (disconnect3d) [inline pwndbg/commands/procinfo.py]: Wondering why those just don't parse a file themselves

Comment (disconnect3d) [reply pwndbg/commands/procinfo.py]: should probably be lib.procfs.net.unix :P

File: pwndbg/commands/procinfo.py [hunk 2]
```diff
@@ -66,12 +67,39 @@
 }
 
 
+class System:
```
Comment (disconnect3d) [inline pwndbg/commands/procinfo.py]: We dont really need a class for this, do we?

**Review by disconnect3d — CHANGES_REQUESTED** [review]
Maybe drop the class

**Comment by patryk4815** [comment]
Done, removed class

**Review by disconnect3d — APPROVED** [review]

**Pull request merged**